src/index.js

```diff
@@ -0,0 +1,55 @@
+let fs = require('fs');
+const prompt = require('prompt-sync')();
+
+
+//analyzes all files in a directory and returns the number of lines for each one
+function lineCounter(){
+	
+	const result = {};
+
+	//asks for user input
+	const path = prompt('Enter the path to the directory to be scanned:     ');
+
+	//reads all files from a given directory
+	let arrayFiles = fs.readdirSync(path);
```

Comment:
Sempre é bom preferir const quando a referência não for ser alterada em nenhum outro ponto do código
```suggestion
  const arrayFiles = fs.readdirSync(path);
```